testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/mdb/dynamic/adapter/TelnetResourceAdapter.java

```diff
@@ -17,15 +17,16 @@
 import jakarta.resource.spi.work.WorkManager;
 import javax.transaction.xa.XAResource;
 import java.io.IOException;
-import java.util.HashMap;
+//import java.util.HashMap;
```

Comment:
Please remove instead of commenting out.

Reply:
Okay! In addition I plan to add null check in the `endpointDeactivation()` method and then unmark draft of this PR.